src/Admin.php

```diff
@@ -39,6 +53,13 @@ public static function init() {
     // Exclude subscribers from post author select options to prevent a performance
     // slowdown on sites with large amounts of non-administrative registered users.
     add_filter('wp_dropdown_users_args', __CLASS__ . '::wp_dropdown_users_args');
+
+    // Limit revisions per post type and age (publishing date).
+    add_filter('wp_revisions_to_keep', __CLASS__ . '::wp_revisions_to_keep', 10, 2);
+    if (!wp_next_scheduled(static::CRON_EVENT_REVISION_CLEANUP)) {
+      wp_schedule_event(time(), 'twicedaily', static::CRON_EVENT_REVISION_CLEANUP);
```

Comment:
Maybe makes sense to also make the `twicedaily` argument filterable as some other instances might not need to run it that often or more often.